src/lang-go.ts

```diff
@@ -274,7 +274,7 @@ function rootURIFromDoc(doc: sourcegraph.TextDocument): URL {
 
 function repoNameFromDoc(doc: sourcegraph.TextDocument): string {
     const url = new URL(doc.uri)
-    return url.pathname.slice(2)
+    return path.join(url.hostname, url.pathname.slice(1))
```

Comment:
Use path.posix.join() - URLs always use forward slashes, and this would use backslashes if you ran tests for this on Windows.